bindings/python/capstone/tricore.py

```diff
@@ -0,0 +1,45 @@
+# Capstone Python bindings, by billow <billow.fun@gmail.com>
+
+import ctypes, copy
+from .tricore_const import *
+
+class TriCoreOpMem(ctypes.Structure):
```

Comment:
and plz capital here

Reply:
> and plz capital here

![image](https://user-images.githubusercontent.com/24983369/234862695-de06fdb4-d109-4b45-a29d-65e5ca0217d0.png)

But I noticed that PowerPC isn't capitalized either. I think there might not be such a convention?

Reply:
Thinking about it a second time I would be against capitalizing it. We would need to add `.upper()` pretty much everywhere in the LLVM backends when using the target name (which is done a lot). So the code gets more unreadable.
@kabeor Is there any advantage for capitalization which would make this worth it?

Reply:
Oh sorry, my bad. Here is good.